v20.2/delete.md

```diff
@@ -108,6 +108,10 @@ To delete a large number of rows (i.e., tens of thousands of rows or more), we r
 
 In the sections below, we provide guidance on batch deleting with the `DELETE` query filter [on an indexed column](#batch-delete-on-an-indexed-column) and [on a non-indexed column](#batch-delete-on-a-non-indexed-column). Filtering on an indexed column is both simpler to implement and more efficient, but adding an index to a table can slow down insertions to the table and may cause bottlenecks. Queries that filter on a non-indexed column must perform at least one full-table scan, a process that takes time proportional to the size of the entire table.
 
+{{site.data.alerts.callout_danger}}
+Exercise caution when batch deleting rows from tables with foreign key constraints and explicit [`ON DELETE` foreign key actions](foreign-key.html#foreign-key-actions) (e.g., [tables that enable interleaved fast path deletes](interleave-in-parent.html#interleaved-fast-path-deletes)). To preserve `DELETE` performance on tables with foreign key actions, we recommend using smaller batch sizes, as additional rows updated or deleted due to `ON DELETE` actions can make batch loops significantly slower, and will not be reflected in the output of a `DELETE` query.
```

Comment:
I think I'd remove the `e.g.` about interleaved tables. 

If I understood @drewdeally 's comment correctly, if you're using interleaved tables and ON DELETE CASCADE, the child table's rows *will* be counted in the DELETE result, so you would see numbers higher than the batch size (I haven't verified this myself). This is basically a bug, but one that can't really be fixed while preserving the fast cascading delete for interleaved tables.

Reply:
Correct, my recommendation is to reflect this in the documentation so the user is not surprised the results.